system/ee/ExpressionEngine/Addons/structure/tab.structure.php

```diff
@@ -718,9 +718,14 @@ public function get_parent_fields($entry_id, $data)
 
         // PARENT BUG
         // Update: Changed this to `$entry_id` which may cause some oddity with the Parent dropdown.
-        $data = $this->sql->get_data($entry_id);
+        // $data = $this->sql->get_data($entry_id);
```

Comment:
We are calling this function in just one place, where $data is present, so I think this is good removal of duplicate query